tests/integ/scala/test_sql_suite.py

```diff
@@ -10,9 +10,12 @@
 from snowflake.snowpark.exceptions import SnowparkSQLException
 from snowflake.snowpark.functions import col
 from snowflake.snowpark.types import LongType, StructField, StructType
-from tests.utils import TestFiles, Utils
+from tests.utils import IS_IN_STORED_PROC, TestFiles, Utils
 
 
+@pytest.mark.skipif(
+    IS_IN_STORED_PROC, reason="SNOW-533647: Support statement level parameter"
```

Comment:
I guess this is already done and we just need to change the tests to avoid using `alter session` to pass in statement level parameter?

Reply:
Yeah we might have already supported this, as long as the statement level parameter used here is allowed in owner's mode. Let's add a TODO and we could look into it later.

Reply:
Hmm I have tried to enable it, but it failed because `alter session` is not allowed in SP. The correct way is to use https://github.com/snowflakedb/snowflake-connector-python/blob/2ccd3c811958170854ff4cac159e7c125bf8b4f3/src/snowflake/connector/cursor.py#L568. (but there is no such API in snowpark). Maybe our `collect()`, `show()` should support this argument. cc @sfc-gh-yixie

Reply:
I created https://snowflakecomputing.atlassian.net/browse/SNOW-609875 to track that whether we can enable more tests.